BlitzBoard/Features/HomeScreenTableCell/TableViewCellViewController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  TableViewCellViewController.swift
+//  BlitzBoard
+//
+//  Created by Kiasha Rangasamy on 2024/04/17.
+//
+
+import UIKit
+
+class StandingTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var positionLabel: UILabel!
```

Comment:
can make these private. Please do for all future PR's

Reply:
Will make the changes